File: templates/python/fr/text/code/stepUp.py
# Fichier: PremierKarel.py
# -----------------------------
# Le programme PremierKarel définit un " main "
# fonction avec des commandes qui poussent Karel à prendre
# a jeton et placez-le sur un rebord.
from karel.stanfordkarel import *

def main():
   avancer()
   ramasser_jeton()
   avancer()
   tourner_gauche()
   avancer()
   tourner_gauche()
   tourner_gauche()
   tourner_gauche()
   avancer()
   avancer()
   laisser_jeton()
   avancer()

File: templates/java/fr/text/code/stepUp.java
/*
 * Fichier: JetonRamasserKarel.java
 * --------------------------------
 * Le programme JetonRamasserKarel définit une méthod
 * e " run " avec des commandes permettant à Karel de
 * prendre un jeton et de le placer sur un rebord.
 */
import stanford.karel.*;
public class JetonRamasserKarel extends Karel {
   public void run() {
      avancer();
      ramasserJeton();
      avancer();
      tournerGauche();
      avancer();
      tournerGauche();
      tournerGauche();
      tournerGauche();
      avancer();
      avancer();
      laisserJeton();
      avancer();
   } 
}